.github/workflows/cron.yml

```diff
@@ -7,8 +7,11 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  cron:
```

Comment:
ooh interesting. this makes sense seeing it now, but I wouldn't have known this without having seen one of these before.